js/constants.js

```diff
@@ -0,0 +1,10 @@
+const initial = {
```

Comment:
Should we use SCREAMING_SNAKE_CASE for these constant global variables?

Reply:
I'm not familiar with the best practices; but we could use it if we stay consistent with it?